File: Swift/Arrays/Easy/FindMaxMin.swift
import Foundation 

func findMaxMin(_ nums: [Int]) {
    if nums.isEmpty{
        print("Array is empty.")
        return 
    }
    var maxElement = nums[0]
    var minElement = nums[0]
    for num
     in nums {
        if num > maxElement{
            maxElement = num 
        }
        if num < minElement {
            minElement = num 
        }
    }
    print("Maximum: \(maxElement)")
    print("Minimum: \(minElement)")
}

let nums = [5, 2, 9, 1, 7]
findMaxMin(nums)

File: Java/Arrays/Easy/FindMaxMin.java
package Java.Arrays.Easy;

public class FindMaxMin {
    public static int[] findMaxMin(int[] nums){
        if (nums.length == 0) {
            System.out.println("Array is empty.");
            return nums ;
        }
        int maxElement = nums[0];
        int minElement = nums[0];

        for (int num : nums) {
            if (num > maxElement) {
                maxElement = num;
            }
            if (num < minElement) {
                minElement = num;
            }
        }
        System.out.println("Maximum: " + maxElement);
        System.out.println("Minimum: " + minElement);
        return nums;
    }
    public static void main(String[] args) {
        int[] nums = {5, 2, 9, 1, 7};
        findMaxMin(nums);
    }
    
}
